docs/user/administration/users/xprofile.md

```diff
@@ -0,0 +1,107 @@
+# BuddyPress xProfile fields administration
```

Comment:
In order to maintain consistency with menu items, tabs, etc., of which should hold precedence, unless there is a good reason to change precedence.
```suggestion
# Profile Fields Screen
```

Reply:
Thanks a lot for your suggestions. About this one, I prefer to leave it the way it is to be consistent with other Documentation page titles.